src/utils/notify/index.js

```diff
@@ -0,0 +1,35 @@
+const admin = require('firebase-admin');
+const { query: Hasura } = require('../hasura');
+const { getFcmToken } = require('./queries');
+
+const notify = (entityTypeId, entityId, actorId, notifierIds) => {
+  const fcmToken = Hasura(getFcmToken, {
+    id: { _eq: notifierIds },
+  }).then((res) => res.json().data);
+
+  new Promise(async (resolve, reject) => {
+    const message = {
+      fcmToken,
```

Comment:
so since notifierIds is an Array could receive multiple tokens. Can you check the documentation of this API call if they support sending an Array for `fcmToken`. I want you to find out how to notify multple targets at once.